### ISSUE TITLE
"Undo" function for 'order complete'

### ISSUE THREAD

**Issue by Kc2693** [Kc2693/FR-DOMShop#2]
When pressing 'order complete' I don't want to have an alert popup for "are you sure?" because that's annoying and so 1990s. Instead I'll have it *visually* delete the order, but keep the order in the db for a certain amount of time (5mins? until user logs out?). 

Either display a smaller "XX user's order has been completed. Undo?" 
User can press undo to bring back the order as it last appeared. 

So tldr:
-> User presses 'order complete'
-> Save all pertinent order for info
-> Remove visually
-> Put in new "Undo?" button in place
-> Clicking undo brings back saved order form
